### DISCUSSION TITLE
Flag proposals with low review score

### DISCUSSION THREAD

**Discussion by crablab** [pretalx/pretalx#1314] [Ideas]
As an acceptance committee, we want to identify any proposals which have received a score of 1, so we can investigate if there are significant issues with the proposal. 

We (MCH) tend to bulk accept high scoring proposals based on average score, but this hides very low scores which can be an indication that someone has specific knowledge about why something should not be accepted. 

We want to see if a proposal has received any 1s, so we can manually review.

**Comment by rixx** [comment]
I'd be happy to merge a PR with this feature!

Implementers should note that defining "low review score" is going to be a hassle. Is it "accumulated score of any one reviewer is below x"? And what is x? Is it "score in any or all categories is below x"? Are independent categories treated separately? And so on.

**Reply by crablab** [reply]
I think this is on a per proposal basis. So in `orga/event/$event/reviews/`, the table should indicate if there are any review scores below a threshold value (which could be configurable). 

It could be implemented as:
- "Number of review scores below X" : int()
- "Proposal has review score below X" : bool()
- "Range" : int() - int()

I think I prefer the last one, although it would be the most complex visually. I am not a statistician, but perhaps what we are actually measuring here is the standard deviation - ie. how spread out the review scores are. 

Do you have any feelings on what you would prefer?

**Reply by rixx** [reply]
Flagging proposals with unusually high standard deviation as "controversial" sounds good.

**Reply by crablab** [reply]
I'll aim to get a PR in for this :)

**Reply by crablab** [reply]
@rixx Hey 👋  Been busy so only just getting around to this! 👍  

Just been getting to grips with Pretalx development (thanks for the great docs and test data generation). I've had a go implementing this in the way I would have found useful: https://github.com/crablab/pretalx/commit/09a779dcae7f8a803ae2a377bc1222ba6e29d8b1

I don't know if you want me to make a PR for feedback, or you'd prefer to discuss here first.

**Reply by rixx** [reply]
I generally like PRs for discussions, but for initial higher level comments either is fine – I'll just drop my vague thoughts here.

First of all, thank you! I like the implementation, and my primary concerns are with the presentation: I'm not sure everybody will know what a standard deviation is and does. This might spook users who are already overwhelmed by the review screen. I can see three ways out of that:

- hide the column unless enabled in the settings (not a fan, settings growth isn't great)
- call it something different (consensus level? can we sensibly map the stdev to a % scale, given that we know what the highest and lowest possible scores are?)
- don't show a number at all, just mark controversial proposals with an icon with a big alt text (big red lightning or something like that)

Do you have any ideas/opinions on these?

Regardless of the chosen solution, I definitely think we should add the stdev as a possible export field to proposal exports, as the people who most enjoy playing with the review data usually already look at the excel export!

**Reply by crablab** [reply]
Hey! 

I'm sorry for going quiet on this - work and life caught up with me. 

I think that's great feedback and thank you for taking the time to look at this. 

On the options: 
- I agree that settings growth isn't great. It also means this feature isn't available by default and people that might find it useful will never turn it on! 
- This is my preferred solution. Std deviation isn't particularly friendly and I think a 'consensus %' could be a nice way of showing this. I think it is doable - I will have a go. 
- So I don't prefer this as it's like an error message that tells you there is a problem...but not what the problem is! Is the consensus below 50%? Or is it below 90%? Perhaps I want to configure the threshold? Is this now cluttering the UI? etc. 

I will also have a look at adding this to the exports! Thanks for the heads up on that.

**Reply by rixx** [reply]
Yeah, I agree with you on most of these – except maybe the last point, as we could include the details on hover and on click (expanding the review rows on-click is something coming soon, anyway), so going with "icon first, details (consensus level or otherwise) second" might be a good option. (Also very much an implementation detail, of course, so not important to figure out rn!)
